test-defaults.ini

```diff
@@ -6,6 +6,9 @@ ws.auth_required = False
 
 is_test_running = True
 
+# Prevent plugins from loading during tests
+plugins_dir = '%(root)s/test_plugins'
```

Comment:
I actually originally had that exact string! And then I was like, "well, maybe we DO want some test_plugins!" But I think you're right (and we have already have tests for loading test plugins).